src/modes/direct_select.js

```diff
@@ -72,8 +72,11 @@ DirectSelect.onFeature = function(state, e) {
 };
 
 DirectSelect.dragFeature = function(state, e, delta) {
-  moveFeatures(this.getSelected(), delta);
-  state.dragMoveLocation = e.lngLat;
+  const isDraggable = (state.feature.type === Constants.geojsonTypes.POINT && state.draggablePoint) || (state.feature.type === Constants.geojsonTypes.LINE_STRING && state.draggableLine) || (state.feature.type === Constants.geojsonTypes.POLYGON && state.draggablePolygon);
```

Comment:
**Suggestion:** Split this into e different variables to avoid having this giant line.
```
const isDraggablePoint = state.feature.type === Constants.geojsonTypes.POINT && state.draggablePoint;
const isDraggableLine = state.feature.type === Constants.geojsonTypes.LINE_STRING && state.draggableLine;
const isDraggablePolygon = state.feature.type === Constants.geojsonTypes.POLYGON && state.draggablePolygon;
const isDraggable = isDraggablePoint || isDraggableLine || isDraggablePolygon;
```